handlers.js

```diff
@@ -415,28 +916,132 @@
+                              let u = -1;
+                              const r = 10**dp;
+
+                              do {
+                                bytes /= thresh;
+                                ++u;
+                              } while (Math.round(Math.abs(bytes) * r) / r >= thresh && u < units.length - 1);
+
+
+                              return bytes.toFixed(dp) + ' ' + units[u];
+                            }
+
+                            var lastModifiedMonth = file.modificationTime.getMonth() + 1
+                            var lastModifiedDay = file.modificationTime.getDate()
+                            var lastModifiedYear = file.modificationTime.getFullYear().toString().substring(2, 4)
+                            var lastModifiedHours = file.modificationTime.getHours() + 1
+                            var lastModifiedMinutes = file.modificationTime.getMinutes() + 1
+                            var lastModifiedSeconds = file.modificationTime.getSeconds() + 1
+
+                            var lastModified = lastModifiedMonth+
+                                               lastModifiedDay+
+                                               lastModifiedYear+
+                                               lastModifiedHours+
+                                               lastModifiedMinutes+
+                                               lastModifiedSeconds
+                            if (lastModifiedSeconds.toString().length != 2) {
+                                var lastModifiedSeconds = '0' + lastModifiedSeconds
+                            }
+                            if (lastModifiedMinutes.toString().length != 2) {
+                                var lastModifiedMinutes = '0' + lastModifiedMinutes
+                            }
+                            if (lastModifiedDay.toString().length != 2) {
+                                var lastModifiedDay = '0' + lastModifiedDay
+                            }
+                            if (lastModifiedHours >= 12) {
+                                var lastModifiedAmPm = 'PM'
+                                if (lastModifiedHours > 12) {
+                                    var lastModifiedHours = lastModifiedHours - 12
+                                }
+                            } else {
+                                var lastModifiedAmPm = 'AM'
+                            }
+                            var lastModifiedStr = lastModifiedMonth+'/'+
+                                                  lastModifiedDay+'/'+
+                                                  lastModifiedYear+', '+
+                                                  lastModifiedHours+':'+
+                                                  lastModifiedMinutes+':'+
+                                                  lastModifiedSeconds +' '+
+                                                  lastModifiedAmPm
```

Comment:
maybe better to just use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat or something

Reply:
Thank you! I will do this